android-app/app/src/main/java/de/dhbw/ka/se/fibo/utils/ApiUtils.java

```diff
@@ -16,12 +15,18 @@
 import java.util.Map;
 import java.util.Optional;
 
+import de.dhbw.ka.se.fibo.BuildConfig;
+
+
 public class ApiUtils {
+    private ApiUtils() {
+        throw new IllegalStateException("Utility class");
+    }
```

Comment:
I'll merge this PR even if codecov detects uncovered lines of code. I double checked the test and can confirm that these lines are correctly covered by [ApiUtilsTest.java](https://github.com/Cebox82/FiBo/blob/cb5aa3f3455ec4b7fa4f8b2f22efb06b3344d3b9/android-app/app/src/test/java/de/dhbw/ka/se/fibo/utils/ApiUtilsTest.java)